Content.Shared/Trigger/Components/Effects/RattleOnTriggerComponent.cs

```diff
@@ -25,6 +25,10 @@ public sealed partial class RattleOnTriggerComponent : BaseXOnTriggerComponent
     public Dictionary<MobState, LocId> Messages = new()
     {
         {MobState.Critical, "rattle-on-trigger-critical-message"},
-        {MobState.Dead, "rattle-on-trigger-dead-message"}
+        {MobState.Dead, "rattle-on-trigger-dead-message"},
+
+        //starlight edit begin
+        {MobState.Alive, "rattle-on-trigger-distress-message"}
+        //starlight edit end 
```

Comment:
please don't add this to every thing with `RattleOnTriggerComponent`

```suggestion
        {MobState.Dead, "rattle-on-trigger-dead-message"}
```